src/intrinsic/mod.rs

```diff
@@ -1006,20 +1027,20 @@ impl<'a, 'gcc, 'tcx> Builder<'a, 'gcc, 'tcx> {
             res.to_rvalue()
         }
         else {
+            assert!(!signed);
```

Comment:
This assert is not needed anymore since you reverted the above condition to `if signed`.

Reply:
Addressed in 5061e3ad163a75e81ed7b69e2113b5db135bc51a